tests/lang/s01_basics/s00_atoms/t01_statement.nim

```diff
@@ -107,34 +143,83 @@ block noreturn_statement:
 
       ## Statement placed before `continue` is executed on every loop iteration
       doAssert preContinue == 0 + 1 + 2 + 3 + 4
-
-
+      ## If you flatten out the for loop; this is what you would see:
+      ## ```nim
+      ## i = 0
+      ## preContinue = preContinue + i
+      ## 
+      ## i = 1
+      ## preContinue = preContinue + i
+      ##
+      ## i = 2
+      ## preContinue = preContinue + i
+      ## 
+      ## i = 3
+      ## preContinue = preContinue + i
+      ## 
+      ## i = 4
+      ## preContinue = preContinue + i
+      ## ```
+      ## 
+      ## If we did not have the `continue` operation; we would instead see this:
+      ## ```nim
+      ## i = 0
+      ## preContinue = preContinue + i
+      ## postContinue = 9
+      ## 
+      ## i = 1
+      ## preContinue = preContinue + i
+      ## postContinue = 9
+      ## 
+      ## i = 2
+      ## preContinue = preContinue + i
+      ## postContinue = 9
+      ## 
+      ## i = 3
+      ## preContinue = preContinue + i
+      ## postContinue = 9
+      ## 
+      ## i = 4
+      ## preContinue = preContinue + i
+      ## postContinue = 9
+      ## ```
```

Comment:
Overly verbose, either reduce number of iterations to the minimal possible value or simplify explanation.